readme.md

```diff
@@ -173,6 +169,14 @@ Once you do a search, you should see a list similar to the one below (versions m
 Please use [GitHub Issues](https://github.com/windows-toolkit/WindowsCommunityToolkit/issues) for bug reports and feature requests.
 For general questions and support, please use [Stack Overflow](https://stackoverflow.com/questions/tagged/windows-community-toolkit) where questions should be tagged with the tag `windows-community-toolkit`.
 
+## <a name="supported"></a> Required Dependencies
+The following dependencies are required for building the Windows Community Toolkit repo and sample app:
+
+* [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/)
+* [Windows SDK May 2019 Update	18362](https://developer.microsoft.com/en-us/windows/downloads/sdk-archive)
```

Comment:
@azchohfi is there a good way to validate this is the only SDK we need? I just tried to build the toolkit again and it wanted me to also install 17134 or something...?

Reply:
Are you sure? It shouldn't. What was the error/warning?

Reply:
Wait, this is wrong... https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/master/Directory.Build.props#L18
Only the sample app targets 18362.

Reply:
```suggestion
* [Windows SDK October 2018 Update 17763](https://developer.microsoft.com/en-us/windows/downloads/sdk-archive)
* [Sample App - Windows SDK May 2019 Update 18362](https://developer.microsoft.com/en-us/windows/downloads/sdk-archive)
```